### ISSUE TITLE
bootstrap.sh Error: aws-sam-cli 1.8.0 is already installed

### ISSUE THREAD

**Issue by deki** [aws-samples/aws-serverless-cicd-workshop#46]
Running bootstrap.sh as described on https://cicd.serverlessworkshops.io/java/setup/bootstrap.html fails with 
```
Error: aws-sam-cli 1.8.0 is already installed
To upgrade to 1.10.0, run `brew upgrade aws/tap/aws-sam-cli`.
```

Workaround is to change line 34 from install to upgrade.

**Comment by ferdingler** [comment]
Thanks for pointing this out @deki . The bootstrap script was written to upgrade AWS SAM to the latest version because Cloud9 used to ship with a very old version. Looks like this is no longer a problem so the bootstrap script may not even be necessary anymore.

**Comment by ferdingler** [comment]
If you think you have the fix. Feel free to submit a PR to the `dev` branch. We can test it on the dev environment http://dev.cicd.serverlessworkshops.io/ before pushing to the official domain name.

**Comment by deki** [comment]
With the applied fix SAM 1.8.0 is used. This is working fine for NodeJS but for Java some folks reported the following error message:
`samcli.local.docker.manager.DockerImagePullFailedException: Could not find amazon/aws-sam-cli-emulation-image-java8:rapid-1.8.0 image locally and failed to pull it from docker.`

We were able to fix that using `brew upgrade aws-sam-cli` with version 1.10.0. So I'll submit another PR.

**Comment by ferdingler** [comment]
Awesome, thank you @deki
